docs/source/lora_without_regret.md

```diff
@@ -0,0 +1,158 @@
+# LoRA Without Regret
```

Comment:
I'd recommend having the code examples first, and then the explanation. I feel it's more actionable this way